src/lib/plugin.js

```diff
@@ -263,12 +284,12 @@ class FiveBellsLedger extends EventEmitter2 {
       this.connection
         .on('connect', (ws) => {
           this.ws = ws
-          this.connected = true
+          this.ready = true
```

Comment:
Why is `ready` only set when the websocket connects? Doesn't it have the metadata before then?